src/Node/FeatureNode.php

```diff
@@ -24,7 +24,7 @@ class FeatureNode implements KeywordNodeInterface, TaggedNodeInterface
     use TaggedNodeTrait;
 
     /**
-     * @param string[] $tags
+     * @param list<string> $tags
```

Comment:
this one is technically a fix for a level 7 error: `getTags` in TaggedNodeInterface requires returning a list, and the readonly promoted property is what gets returned, so we need to expect the same in the constructor (which is quite fine as the Parser produces a list anyway)